README.md

```diff
@@ -1,15 +1,59 @@
 # hello-hydra
 
-An example eosio.token contract tested with [Hydra](docs.klevoya.com/hydra).
+[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)
 
-To install the dependencies run:
+[![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+Hydra is an EOSIO smart contract testing framework that allows you to quickly get started with testing your smart contract code. Besides Hydra, installing additional software or running your own local blockchain node is not required. 
+
+# New Features!
+
+  - Import a HTML file and watch it magically convert to Markdown
+
+You can also:
+  - Import and save files from GitHub, Dropbox, Google Drive and One Drive
+
+### Tech
+
+Hydra uses a number of open source projects to work properly:
+
+* [Jest] - a delightful javascript testing framework maintained by facebook with a focus on simplicity
+
+### Installation
+
+Hydra requires [Node.js](https://nodejs.org/) v10.16.0 to run.
```

Comment:
It should work fine with any recent Nodejs version. Also with Node 12. I just set it to Node 10, because Node 8 is not supported anymore.

So I'd say something like this instead:
Hydra requires a recent Node.js version, v10.16.0 or later.

Reply:
If it doesn't work with Node 12 for you, please let me know, we should definitely fix that.